src/components/Card/LandingPageCard/LandingPageCard.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import propTypes from 'prop-types';
+import { motion } from 'framer-motion';
```

Comment:
did you make changes to the landing page card in this branch or these are old changes?

Reply:
Yes i have used the primary color and changed animation to Framer Motion

Reply:
you should have changed these in the landingPage branch and pushed, however will check and merge them for now but in. the future make the changes in their branch its better to organize everything